package.json

```diff
@@ -34,7 +34,7 @@
     "@tanstack/react-query": "4.29.7",
     "@tanstack/react-table": "^8.9.1",
     "classnames": "^2.3.2",
-    "design-system-react": "https://github.com/cfpb/design-system-react",
+    "design-system-react": "https://github.com/cfpb/design-system-react#117-react-router-links",
```

Comment:
This line should be reverted [once this PR](https://github.com/cfpb/design-system-react/pull/294) is merged.